File: Administration Console/src/SpottedAdminConsole/Models/WantedPerson.cs
namespace SpottedAdminConsole.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class WantedPerson : Person
    {
        [MaxLength(2048)]
        public string Description { get; set; }

        public DateTime Date { get; set; }

        public string Picture { get; set; }
    }
}

File: SpottedApp/app/src/main/java/com/dirtybits/spotted/Models/WantedPerson.java
package com.dirtybits.spotted.Models;

import android.databinding.Bindable;
import android.media.Image;
import com.dirtybits.spotted.BR;
import java.util.Date;

public class WantedPerson extends Person {

    private static int id = 0;

    private Date wantedDate;

    public WantedPerson(String fullName, String description, Image photograph, int timesSpotted, Date wantedDate)
    {
        super(fullName, description, photograph, timesSpotted);
        this.setPersonId(Integer.toString(++id));
        this.setType(Type.WantedPerson);
        this.wantedDate = wantedDate;
    }

    public void setWantedDate(Date wantedDate){
        this.wantedDate = wantedDate;
        notifyPropertyChanged(BR.missingDate);
    }

    @Bindable
    public Date getWantedDate(){
        return this.wantedDate;
    }
}
